cylc/flow/network/client_factory.py

```diff
@@ -16,6 +16,10 @@
 
 from enum import Enum
 import os
+from typing import TYPE_CHECKING, Union
+
+if TYPE_CHECKING:
+    from cylc.flow.network.client import WorkflowRuntimeClientBase
 
 
 class CommsMeth(Enum):
```

Comment:
Soon I'll have an `https` method to add here 👍  ... (going via UIS if it exists (or make-it-exist/start-it-up (?)))